docs/tutorial.md

```diff
@@ -0,0 +1,372 @@
+
+Thie method simply creates the Deployment as specified below.
+
+```
+    private Deployment createMemcachedDeployment(Memcached m) {
+        return new DeploymentBuilder()
+            .withMetadata(
+                new ObjectMetaBuilder()
+                    .withName(m.getMetadata().getName())
+                    .withNamespace(m.getMetadata().getNamespace())
+                    .withOwnerReferences(
+                        new OwnerReferenceBuilder()
+                            .withApiVersion("v1alpha1")
+                            .withKind("Memcached")
+                            .withName(m.getMetadata().getName())
+                            .withUid(m.getMetadata().getUid())
+                            .build())
+                    .build())
+            .withSpec(
+                new DeploymentSpecBuilder()
+                    .withReplicas(m.getSpec().getSize())
+                    .withSelector(
+                        new LabelSelectorBuilder().withMatchLabels(labelsForMemcached(m)).build())
+                    .withTemplate(
+                        new PodTemplateSpecBuilder()
+                            .withMetadata(
+                                new ObjectMetaBuilder().withLabels(labelsForMemcached(m)).build())
+                            .withSpec(
+                                new PodSpecBuilder()
+                                    .withContainers(
+                                        new ContainerBuilder()
+                                            .withImage("memcached:1.4.36-alpine")
+                                            .withName("memcached")
+                                            .withCommand("memcached", "-m=64", "-o", "modern", "-v")
+                                            .withPorts(
+                                                new ContainerPortBuilder()
+                                                    .withContainerPort(11211)
+                                                    .withName("memcached")
+                                                    .build())
+                                            .build())
+                                    .build())
+                            .build())
+                    .build())
+            .build();
+    }
+```
+
+### deleteResource
+
+The delete Resource is an implemented method. The deltion part was take care by the Java Operator SDK that's why it does not have any code as such.
+
+## Run the Operator Locally
+
+### 1. Run locally outside the cluster
+
+The following steps will show how to deploy the operator on the Cluster. However, to run locally for development purposes and outside of a Cluster use the target `make install run`.
+
+First, create CR and CRD's within the cluster.
+
+`kubectl apply -f cr.yaml`
+
+
+`kubectl apply -f memcached-sample.yaml`
+
+Then, Open the entire project with `IntelliJ IDEA` and run the application. We are still looking to run this code with the Container Image.
```

Comment:
Is there no way to run it outside of IntelliJ? You can't assume they have this. What if they are Eclipse users? or use VSCode? I prefer to assume they do not have this IDE and have to run it by hand.

Reply:
I am trying to run it via terminal but it was failing. I am working on it.

Reply:
It is working now and I will push new changes and steps with upcoming commit.